Mathlib/Algebra/Group/Pi/Basic.lean

```diff
@@ -548,6 +548,20 @@ theorem bijective_pi_map {F : ∀ i, f i → g i} (hF : ∀ i, Bijective (F i))
   ⟨injective_pi_map fun i => (hF i).injective, surjective_pi_map fun i => (hF i).surjective⟩
 #align function.bijective_pi_map Function.bijective_pi_map
 
+lemma comp_eq_const_iff (b : β) (f : α → β) {g : β → γ} (hg : Injective g) :
+    g ∘ f = Function.const _ (g b) ↔ f = Function.const _ b :=
+  hg.comp_left.eq_iff' rfl
```

Comment:
```suggestion
    g ∘ f = const _ (g b) ↔ f = const _ b := hg.comp_left.eq_iff
```

Reply:
I don't know what to do about these comments, now that its been merged? make a new pr?

Reply:
Yeah that would be great, thank you!